lib/Algorithm/ConsistentHash/Ketama.pm

```diff
@@ -86,6 +108,30 @@ recomputing the hash.
 Given a number, returns the label associated with that hash number. Only
 hashes returned by hash_with_hashnum are permissible.
 
+=head1 IMPORTANT NOTES FOR USERS OF 0.00009 AND BELOW
```

Comment:
Ah, this is a doc fumble alright, but I just did the change w/o bumping the version numbers...